catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/MetacardApplication.java

```diff
@@ -387,9 +395,11 @@ public void init() {
     post(
         "/subscribe/:id",
         (req, res) -> {
+          String userid = getSubjectIdentifier();
           String email = getSubjectEmail();
```

Comment:
❓ Can we use the unique identifier for workspace operations too? This `getSubjectEmail` method expects the user's email to have an attribute key of `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress`, which may not be the case. If it were configurable (i.e. using the unique identifier), that wouldn't be a problem.

Reply:
The workspace subscription stuff is email only, so I didn't replace that logic here with the unique ID as that may not return email. Emails should be in `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress` claim.